packages/core/lib/use-puck.ts

```diff
@@ -13,6 +15,14 @@ export const usePuck = () => {
     appState,
     config,
     dispatch,
+    getPermissions: (permissions: string[]) => {
```

Comment:
I think we should tweak this API a bit:

1. Drop the `permissions` param and just return all permissions by default
2. Accept an optional `itemSelector` param in case the user wants to get permissions for a non-selected item

Reply:
I changed this to componentType as that was all that was required by the logic.